src/components/spatial/tiling-utils.js

```diff
@@ -1,15 +1,16 @@
+
+export function tileToBoundingBox(x, y, z, maxHeight, maxWidth, tileSize) {
+  return {
+    west: (x * tileSize) * (2 ** (-1 * z)),
+    north: (y * tileSize) * (2 ** (-1 * z)),
+    east: Math.min(maxWidth, ((x + 1) * tileSize) * (2 ** (-1 * z))),
+    south: Math.min(maxHeight, ((y + 1) * tileSize) * (2 ** (-1 * z))),
+  };
+}
+
 /* eslint-disable no-underscore-dangle */
 /* eslint-disable class-methods-use-this */
 
-/*
-------------------------------------------
-THIS CODE IS FROM DECKGL MOSTLY
-IT IS AWAITING A PR THERE AND WILL
-THEN BE REMOVED IN FAVOR OF THAT
-------------------------------------------
-*/
```

Comment:
If we're still waiting, keep the message? Or if we're not waiting, should be able to delete code?

Reply:
This code needs to exist since the PR there only allows you to pass in whatever tiling logic you like.  This allows for geo-spatial and non-geospatial use cases (or anything) without dealing with updating deckgl directly.